package.json

```
@@ -5,7 +5,7 @@
   "main": "discordas.js",
   "scripts": {
     "test": "mocha -r ts-node/register test/config.ts test/test_*.ts test/**/test_*.ts",
```

```suggestion
    "test": "mocha -r ts-node/register test/config.ts 'test/**/test_*.ts'",
```
With properly quoted globs, we can simplify this command too.